network-api/networkapi/mozfest/templates/fragments/carousel/carousel_navigation.html

```diff
@@ -1,4 +1,4 @@
-<div class="swiper-navigation tw-container tx-p-0 tw-relative tw-z-10 tw-mb-12 tw-ml-auto tw-hidden large:tw-flex large:tw-justify-end tw-items-center {% if page.signup != None %} tw-ml-0 tw-container{% endif %}">
+<div class="swiper-navigation tw-container tx-p-0 tw-relative tw-z-10 tw-mb-12 tw-ml-auto tw-hidden large:tw-flex large:tw-justify-end tw-items-center {% if page.signup != None %} tw-ml-0 tw-container{% endif %} [body.mozfest_&]:tw-p-0 [body.mozfest_&]:tw-mb-0">
```

Comment:
Since this component is only used in MozFest and the template is already in a `mozfest` directory, do we need this variant here? Seems like we could apply the classes directly.

Reply:
It's included in `foundation_slider_block.html` (https://github.com/MozillaFoundation/foundation.mozilla.org/blob/main/network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/foundation_slider_block.html#L11) which doesn't look like it's part of the MozFest directory.

Reply:
Oh. Can we create a follow-up ticket to move the carousel code out of the `mozfest` app then? Seems weird to have that reused in a wider applied template/block.